brochures/index.html

```diff
@@ -16,12 +16,8 @@ <h1> Brochures </h1>
 		<a href="{{site.baseurl}}/brochures/Subnero-Multi-Channel-Modems.pdf" target="_blank" style="font-size: 1.2em;">Subnero Multi-channel Underwater Modems</a>
 	</div>
 	<div class="brochure-container">
-		<a href="{{site.baseurl}}/brochures/subnero-swan.pdf"><img class="brochure-thumb" src="swan.jpg"></a>
-		<a  href="{{site.baseurl}}/brochures/subnero-swan.pdf" target="_blank" style="font-size: 1.2em;">Subnero Water Assesment Network (SWAN)</a>
-	</div>
-	<div class="brochure-container">
-	  <a href="{{site.baseurl}}/brochures/SWAN_Brochure_Chinese_v3.1.pdf"><img class="brochure-thumb" src="{{site.baseurl}}/brochures/swan.jpg"></a>
-	  <a  href="{{site.baseurl}}/brochures/SWAN_Brochure_Chinese_v3.1.pdf" target="_blank" style="font-size: 1.2em;">Subnero Water Assessment Network (SWAN) (Mandarin)</a>
+		<a href="{{site.baseurl}}/brochures/SWAN_Brochure.pdf"><img class="brochure-thumb" src="swan_brochure.jpg"></a>
```

Comment:
If the thumbnail image is new, can you delete the old one?

Reply:
Removed the old SWAN brochure and thumbnail.